apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -27,7 +26,7 @@ const CurriculumCatalog = ({
     useState(false);
   const [expandedCardKey, setExpandedCardKey] = useState(null);
 
-  const isQuickViewDisplayed = queryParams()['quick_view'] === 'true';
+  const isQuickViewDisplayed = true;
```

Comment:
Can we just remove uses of `isQuickViewDisplayed` alltogether if we're just showing the expanded card now?